mappings/net/minecraft/client/font/TextRenderer.mapping

```diff
@@ -65,9 +75,15 @@ CLASS net/minecraft/class_327 net/minecraft/client/font/TextRenderer
 		ARG 9 underlineColor
 		ARG 10 light
 	METHOD method_1726 isRightToLeft ()Z
+		COMMENT Checks if the current language the text renderer is can render uses right to left writing.
```

Comment:
I don't understand the phrasing here

Reply:
`Checks if the language being used by the renderer uses right to left` might be better?